README.md

```diff
@@ -48,3 +48,63 @@ Requests to cross-scheme subresources will no longer send SameSite=Lax or SameSi
+
+#### TL;DR
+"Schemeful Same-Site" affects the browsing context cookies will be sent/set in. "Is this url the same site as the one I'm on if I consider the schemes?"
+
+“Scheme-Bound Cookies” affects the connections the cookies will be sent on. "Does the scheme of the request url match the scheme of the original response url that set this cookie?"
+
+### Example
+This example showcases that it's possible to have a situation in which the cookies that "Schemeful Same-Site" and "Scheme-Bound Cookies" each send can differ.
+
+1. A user navigates to https://website.example which contains some embedded resources which want to set and read cookies.
+
+   * A response to the request for https://website.example/setsacookie.jpg has `Set-Cookie: mycookie=val; SameSite=Strict`
+
+2. The user then navigates to http://website.example which is the same page as before, served over an insecure connection.
+
+3. The browser checks to see if it can send the cookie to http://website.example/readsacookie.jpg
+
+   * In the current world (i.e. without either proposal) this cookie is allowed to be sent as this is a same-site context: only the registrable domain matters and they match.
+
+   * “Schemeful Same-Site” would also allow this cookie to be sent as we're still in a same-site context when considering scheme: the user is on http://website.example and the cookie wants to be sent to http://website.example
+
+   * “Scheme-Bound Cookies” would not allow this cookie to be sent as the connection isn't the same as the one the cookie was set by: the cookie was set on https and is trying to be sent on http.
+
+4. Immediately after the browser checks to see if it can send the cookie to https://website.example/readsacookie2.jpg
+
+   * In the current world the cookie is allowed to be sent as this is a same-site context: the registrable domains match.
+
+   * “Schemeful Same-Site” would not allow this cookie to be sent as we're now in a cross-site context: the user is on http://website.example and the cookie wants to be sent to https://website.example.
+
+   * “Scheme-Bound Cookies” would allow this cookie to be sent as the connection is the same as the one the cookie was set by: the cookie was set on https and is trying to be sent on https.
+
+5. Finally the user navigates to a different website, https://othersite.example, which embeddeds https://website.example/readsacookie2.jpg
+
+   * In the current world the cookie not allowed to be sent as this is a cross-site context: the registrable domains don't match
+   
+   * “Schemeful Same-Site” would not allow this cookie to be sent as this is a cross-site context: the registrable domains don't match.
+   
+   * "Scheme-Bound Cookies" would allow this cookie to be sent the connection is the same as the one the cookie was set by: the cookie was set on https and is trying to be sent on https.
```

Comment:
This is confusing because the cookie is set to SameSite=Strict; so scheme-bound cookies is essentially a no-op (i.e. the SameSite attribute overrides it). Should we consider adding another SameSite=None cookie to the example, or would that complicate things too much? Alternatively, we can try to rephrase this bullet.

Reply:
It also reads like scheme-bound cookies is changing this to make the cookie allowed, which isn't right.

Reply:
Hm yeah, maybe split it into two cookies, one SameSite=Strict and another SameSite=None? Maybe that's getting too complicated. I think this whole example (list item 5) could also just be omitted.

Reply:
Yeah, while I knew what I meant, I can see how this could easily be confusioning.
I'm thinking I'll parenthetically mention that while scheme bound doesn't mind this cookie being sent it would still be blocked by same-site.